### ISSUE TITLE
Fix needed at least for IE10 and 11.

### ISSUE THREAD

**Issue by freeads777** [wysiwygjs/wysiwyg.js#51]
Trying to make it work on one of my sites. Works fine in most browsers. And although it says on your page that it works with IE6+ it actually doesn't. 

I don't really care if it works in IE 8 or even 9, but 10 or 11 is kinda important.

IE 10 and 11 bug: Everything needs to be double-clicked for functions to work (bold, italic, etc.). Headers 1, 2, 3, etc. stay selected. There's no way to get out of them once selected.

Is there a fix for IE 10 and 11?

Will be greatly appreciated if you could let me know what I can do to somehow make it work. I spent the last two days trying to make it work and then I went to IE to verify if it works there too and I was like "HOLY MOLLY!" :) Will appreciate any assistance from you.

**Comment by wysiwygjs** [comment]
Works for me (tested with IE11) using this examples:
http://wysiwygjs.github.io/

**Comment by freeads777** [comment]
I tested it in on two different laptops with IE11 and it has the bug. Let me make sure you know what I mean by double click. Let's say if I want it to render Bold if I just click one time nothing happens. If I click twice (like I'd click on an icon) then it begins to render. To cancel it I have to do the same. I can't believe it's not doing it for you... Strange. Like I said I tested it on two different laptops. What can be the reason I am seeing this issue? How can I get rid of that double click issue?
